ee/server/calendar/google_calendar_load.go

```diff
@@ -123,6 +124,10 @@ func (lowLevelAPI *GoogleCalendarLoadAPI) CreateEvent(event *calendar.Event) (*c
 	return &rspEvent, nil
 }
 
+func (lowLevelAPI *GoogleCalendarLoadAPI) UpdateEvent(event *calendar.Event) (*calendar.Event, error) {
+	return nil, errors.New("GoogleCalendarLoadAPI.UpdateEvent is not implemented")
```

Comment:
Should this be implemented, or okay to leave as is?

Reply:
Yes, OK. We are not using this code right now. There is another eng-initiated story to create a dummy Google Calendar server for load tests.